### ISSUE TITLE
Windows

### ISSUE THREAD

**Issue by actraiser30** [procount/pinn#547]
Hello there
can you add more windows in pinn?

Windows 3.1/95/2000/XP/7/10

maybe i did not make my self clear i mean like the windows98 you have for the Raspberry Pi
windows 3.1/windows95/windows 2000/windows XP /windows 10 for pinns image for the raspberry pi

thanks

**Comment by lurch** [comment]
Sorry for the thumbs-down reaction (which I've now removed), I assumed that this would be a non-starter for copyright reasons. But I didn't actually realise that PINN already includes [Windows98](https://sourceforge.net/projects/pinn/files/os/win98-pi/)!

**Comment by actraiser30** [comment]
No problem
About the copyright thing you can just add the windows installers and
people can just add their own legit cd key asked in installation

Στις Τετ, 11 Αυγ 2021, 17:55 ο χρήστης Andrew Scheller <
***@***.***> έγραψε:

**Comment by actraiser30** [comment]
I forgot win98 runs in qemu and its very slow and buggy,it needs to run
from dosbox
Thanks

Στις Τετ, 11 Αυγ 2021, 19:00 ο χρήστης G D ***@***.***> έγραψε:

**Comment by procount** [comment]
I don't have time to convert images from qemu to dosbox I'm afraid. Unless it's a simple conversion from an image file that already exists (and I would need a URL for that) to PINN/NOOBs format, I doubt it will get done.

**Comment by actraiser30** [comment]
is it possible to make the img file/partition of windows 2GB or 4GB its only 256mb and i cant install some games to try
or tell me how can i do it my self i understand you dont have time for this things
thanks

**Comment by lurch** [comment]
Does https://pinn.mjh.nz/ help?

**Comment by procount** [comment]
Hmm.
I don't think Matt's webpage will help, because it only expands the "flexible" partitions.
IIRC, Windows on the Pi uses about 3-4 partitions. I imagine the 1st boot partition is only 256MB, but this should be sufficient for its purpose. I don't think this is where you should be installing games - Are you just trying to copy the game to the SD card whilst it is inserted in a PC?

**Comment by actraiser30** [comment]
Yes,but the 256mb img partition of windows its not enough,windows are
installed in an img file inside rapbian os that loads with qemu,and that
img file is 256mb with less that 100mb left in the windows.img file not in
the partition, thats the file i want to expand.not the raspbian partition.

Στις Δευ, 23 Αυγ 2021, 14:30 ο χρήστης procount ***@***.***>
έγραψε:

**Comment by procount** [comment]
Maybe I was thinking of a different implementation. Sorry, my knowledge is not enough in this area.

**Comment by lurch** [comment]
It sounds like that's something you ought to be asking to wherever/whoever you downloaded your "Windows image" from (and/or check the qemu and/or dosbox documentation - maybe you can add extra "virtual drives" to the emulation?), rather than the PINN developer?

**Comment by actraiser30** [comment]
No,problem,i will figure it out.

Even windows 11 work in Rpi

Thanks a lot

Στις Δευ, 23 Αυγ 2021, 15:04 ο χρήστης procount ***@***.***>
έγραψε:

**Comment by actraiser30** [comment]
I did not download the image anywere it was included in pinns,otherwise i
wont be asking anything.read the first post before you "make up things"

Στις Δευ, 23 Αυγ 2021, 15:11 ο χρήστης Andrew Scheller <
***@***.***> έγραψε:

**Comment by lurch** [comment]
Sorry, I wasn't "making up things", but I've never run (any version of) Windows on a Pi, so I didn't know how it worked.
You previously talked about both qemu and dosbox, and in a recent comment you said "windows are installed in an img file inside rapbian os that loads with qemu" so I assumed that this was an img file that you had downloaded yourself - I had no idea that the Win98 OS in PINN actually installed Raspbian.

**Comment by actraiser30** [comment]
No problem,pinns rocks anyway

Στις Δευ, 23 Αυγ 2021, 16:01 ο χρήστης Andrew Scheller <
***@***.***> έγραψε:

**Comment by JeromTWinL** [comment]
@actraiser30 at your windows list, only windows 10 is supported as full system to run on Raspberry Pi 3/4/400
